app/script_variant.py

```diff
@@ -31,9 +31,7 @@ def read_dictionary(file_name):
         return None
     return dictionary
 
-JAPANESE_NAME_DICTIONARY = read_dictionary('japanese_name_dict.txt')
```

Comment:
Can you delete japanese_name_dict.txt and jp_location_dict.txt which are no longer used?